controllers/rhmi/bootstrapReconciler.go

```diff
@@ -588,6 +595,14 @@ func (r *Reconciler) reconcileOauthSecrets(ctx context.Context, serverClient k8s
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
 
+func (r *Reconciler) reconcileAddonManagedApiServiceParameters(ctx context.Context, serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
+	_, err := addon.GetAddonParametersSecret(ctx, serverClient, r.ConfigManager.GetOperatorNamespace())
+	if k8serr.IsNotFound(err) && err != nil {
```

Comment:
Is it just me or there is something weird here. So we want to fail when there is an error and this is a `not found` error. Why not just fail on `not found`? :thinking: or on any error?